### DISCUSSION TITLE
Is there no commemoratio of S. Thomæ Apostoli in the vespers of 12-21-2024? 1960 rubrics

### DISCUSSION THREAD

**Discussion by youakrim** [DivinumOfficium/divinum-officium#4212] [Q&A] [answered]
Hello,
The title of vespers for 12-21-2024 in 1960 rubrics states :
```
Dominica IV Adventus ~ I. classis
Commemoratio: S. Thomæ Apostoli
Vespera de sequenti; commemoratio de præcedenti
```
I see no commemoratio after the oratio, is it normal ?

**Comment by APMarcello3** [comment, marked as answer]
Per the table of concurrence, the office displays correctly but the title needs to be changed.

@FAJ-Munich 

![Concurrence table 1960.jpg](https://github.com/user-attachments/assets/31003add-f2fe-45ac-b437-91e32f5f82a4)

**Reply by youakrim** [reply]
Thank you!
